### DISCUSSION TITLE
Pause other audio players in browser window when another played

### DISCUSSION THREAD

**Discussion by jacobbinnie** [sampotts/plyr#2219] [Q&A]
Hey there, I've embedded a Plyr audio player as a CMS item within my site. As I've got various CMS items populating with multiple audio tracks, currently a user could click play on them all and start a choir within the window.

What can I do to pause all other plyr players when another is played?

**Comment by aubreyz** [comment]
I have really been battling with this - a fairly major problem if more than one player per page. Nothing I do works consistently (including several code snippets folk have proposed in the issues). Any hints would be much appreciated.
